modules/store/src/store.ts

```diff
@@ -109,11 +109,13 @@ export class Store<T> extends Observable<T> implements Observer<Action> {
 
 export const STORE_PROVIDERS: Provider[] = [Store];
 
-export function select<T, K>(
-  mapFn: (state: T) => K
-): (source$: Observable<T>) => Observable<K>;
+export function select<State, Result, Props>(
```

Comment:
Leave the types as `T, K` in this PR. Its confusing to switch between `T, K` and `State, Result`